ui/app/upload_image/controller.js

```diff
@@ -15,11 +15,17 @@ const uploadFile = multer({
 const handleUploadImageRoute = [
     uploadFile.single('file'),
     (request, response, next) => {
-        const file = request.file
         const token = request.session.jwt
-        handlePostingImageToAPI(file, token).then((result) => {
-            if (result['sloth_check'].contains_sloth) {
-                response.status(200).render('index', {page: 'Slothbucket', contains_sloth: 'Contains a sloth!'})
+        const file = request.file
+        handleUploadingImage(file, token).then((image) => {
```

Comment:
Rather than call the response object `image`, it would make more sense to call it `uploadResponse` or something